.github/workflows/dev_fe_build_and_deploy.yml

```diff
@@ -47,14 +47,17 @@ jobs:
           context: ${{ github.workspace }}/${{ env.WORKING_DIR }}
           dockerfile: ${{ github.workspace }}/${{ env.WORKING_DIR }}/${{ env.DOCKER_FILE }}
           image_tags: "unstable,${{ github.sha }},${{ steps.extract_version.outputs.version }}"
+          build_args: |
```

Comment:
So we already are specifying env vars within terraform too.... is it desired to be specifying them here as well ?

Reply:
The `build_args` are used for building the container image before Terraform gets involved.